lib/index.js

```diff
@@ -63,7 +63,7 @@ Object.defineProperties(Video, {
   },
   runPreflight: {
     enumerable: true,
-    value: require('../es5/preflight').runPreflight
+    value: require('../es5/preflight/preflighttest').runPreflight
```

Comment:
With the new TS Tooling changes, we don't need to reference es5 folder here.

Reply:
yep, Thanks, I will remove `es5`

Reply:
looks like we have to keep this `es5` for now - integration tests load this version (and not es5) version - so it needs to go into es5 for the generated file. We could modify all integration tests to load the es5 version directly, but we can keep it for the next iteration when we update this file to be `.ts`